src/DynamoCore/Scheduler/UpdateGraphAsyncTask.cs

```diff
@@ -180,23 +180,15 @@ private bool Contains(UpdateGraphAsyncTask other)
                    other.graphSyncData.DeletedNodeIDs.All(graphSyncData.DeletedNodeIDs.Contains);
         }
 
-        private bool IsScheduledAfter(UpdateGraphAsyncTask other)
-        {
-            return CreationTime > other.CreationTime;
-        }
-
         protected override TaskMergeInstruction CanMergeWithCore(AsyncTask otherTask)
         {
-            var theOtherTask = otherTask as UpdateGraphAsyncTask;
```

Comment:
I think this change may have significant performance impacts - we should investigate the original reason for the implementation - my memory is that when moving a slider it kicked off hundreds of unnecessary runs and made dynamo unresponsive - if this is related then theres an argument that the crash is actually acceptable...?

Reply:
Strange that was fixed by doing this instead of throttling the requests queued by the slider 🤔 

Still, maybe we could restore a more elaborate merge strategy that accounted for this case in particular. What kind of `graphSyncData` would we expect? Do we have a test case?

Reply:
heres part of the discussion I recall: 
https://github.com/DynamoDS/Dynamo/pull/4276

Reply:
and a related github issue:
https://github.com/DynamoDS/Dynamo/issues/4229

Reply:
`Delay` seems to work just fine according to my manual testing. Is there any special validation you would like to make beyond that issue?

Reply:
I'm concerned if there are other scenarios (other than the slider) where we are not dropping redundant tasks that are quickly accumulating as a result of this change thus hitting performance.

Reply:
We don't control the UI that other package authors may build - delay for the slider is great, but it does not cover the case generally, though I guess we could document the idea somewhere to throttle their controls?

For example I am thinking of controls like:
![image](https://user-images.githubusercontent.com/508936/90567351-05ed8a00-e178-11ea-98e7-576e7c53956d.png)

and

![image](https://user-images.githubusercontent.com/508936/90567476-33d2ce80-e178-11ea-9e10-83a7bab2fb6c.png)

Reply:
Granted. For those cases having a strategy to merge/discard would be useful. I'll try to come up with something.